qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -0,0 +1,53 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+# This is the only folder in qiita_pet that should import outside qiita_pet
```

Comment:
Not sure if we need this warning in every file of just in __init__.py, what do other think?